src/tools/cephfs/tox.ini

```diff
@@ -0,0 +1,8 @@
+[tox]
+envlist = py3 flake8
+skipsdist = true
+
+[testenv:flake8]
+deps = flake8
+passenv = *
+commands = flake8 {posargs:cephfs-shell}
```

Comment:
why would you want to use positional argument for the argument of `flake8`?
i think
```
commands=flake8 ceph_brag.py
```
is good enough.

Reply:
Okay. I will change it.